app/views/solidus_paypal_commerce_platform/admin/payment_methods/_available_to_users.html.erb

```diff
@@ -0,0 +1,8 @@
+<% if @payment_method.type == "SolidusPaypalCommercePlatform::PaymentMethod" && !@payment_method.preferences[:paypal_email_confirmed] %>
+  <%= f.check_box :available_to_users, disabled: true %>
+  <span class="hint-tooltip" title="" data-content="You must verify that your email is confirmed with PayPal before allowing users to check out.">
+    <i class="fa fa-info-circle"></i>
+  </span>
```

Comment:
Would be nice to fully support translations with

```suggestion
  <%= f.field_hint :available_to_users %>
```
plus adding `spree.hints.solidus_paypal_commerce_platform/payment_method.available_to_users` to `locales.yml`

Reply:
Of course, I always forget the translations. Also, I had no idea `f.field_hint` was a thing, that's seriously cool. Thanks!